ENH - Nebari landing page copy (#167)
nebari-dev/nebari-docs: docs/src/pages/index.mdx

---
title: Nebari Homepage
---

<!-- Copy for Nebari Landing page. See Firma file: https://www.figma.com/file/ccOw2sxgjpEUjrOgKqVXRq/Nebari-website?node-id=809%3A3439 -->

## Hero

> Tagline: An opinionated open source platform for Data Science and research teams. Built for scale, designed for collaboration.

Reviewer: I LOVE the second sentence, I think it condenses the Nebari value prop beautifully.

I'd prefer to keep the tagline minimal. I think both "opinionated" and "open source" can be a part of the next section "Why choose Nebari?" -- what do your think?

How about:

> A data science platform: Built for scale, designed for collaboration.

> Tagline: An opinionated open source platform for Data Science and research teams. Built for scale, designed for collaboration.

Author: 🤔  the latter sounds a bit impersonal... what about 

> Your open source Data Science platform: Built for scale, designed for collaboration.

(I know we mention open source later - *but* it never hurts having this front, left, and centre)

> Tagline: An opinionated open source platform for Data Science and research teams. Built for scale, designed for collaboration.

Reviewer: I like it! :)

> Tagline: An opinionated open source platform for Data Science and research teams. Built for scale, designed for collaboration.

Author: ```suggestion
Tagline: Your open source data science platform. Built for scale, designed for collaboration.
```
Changed!

## Why choose Nebari?

> Nebari provides a unified platform for all users to collaborate on data science and research projects.

Reviewer: I think we can keep it all lowercase? I say this mainly based on [Wikipedia](https://en.wikipedia.org/wiki/Data_science) and [this English stack exchange comment with references](https://english.stackexchange.com/a/223162).

It was interesting that the Google and Microsoft style guides didn't include a capitalization reference for data science (neither did [the few other style guides I looked at](https://www.writethedocs.org/guide/writing/style-guides/)).

> Nebari provides a unified platform for all users to collaborate on data science and research projects.

Author: ok let's not capitalise then

It is designed to be flexible and extensible, and built entirely on open source infrastructure.
Nebari is vendor-and-cloud-agnostic, and is designed and developed by and for the data science community.

<!-- Need to highlight 6 features -->
<!-- Added here left-to-right, top-to-bottom -->

1. GitOps approach
   Integrated DevOps best practices for infrastructure management and deployment.
1. Community driven
1. Collaboration first
1. Open source/Upstream focused
   We ensure that the improvemebts we make are contributed back to the open source projects we use.
1. MLOps enabled
   @trallard: not sure about this...
1. Extensible/customisable
   We leverage open source infrastructure and tools in our ecosystem to ensure that you can customise your deployment to your needs.
1. Reproducible environments

## Deploy anywhere

## Integrations

> Nebari supports multiple integrations with tools
> in the open source Data Science ecosystem out of the box.

Author: ```suggestion
Nebari comes with out-of-the-box integrations to multiple tools in the 
Data Science ecosystem.
```

---
title: Nebari Homepage
---

<!-- Copy for Nebari Landing page. See Firma file: https://www.figma.com/file/ccOw2sxgjpEUjrOgKqVXRq/Nebari-website?node-id=809%3A3439 -->

## Hero

Tagline: An opinionated open sourc eplatform for Data Science and research teams. Built for scale, designed for collaboration.

## Why choose Nebari?

Nebari provides a unified platform for all users to collaborate on data science and research projects.
It is designed to be flexible and extensible, and built entirely on open source infrastructure.
Nebari is vendor-and-cloud-agnostic, and is designed and developed by and for the data science community.

<!-- Need to highlight 6 features -->

> <!-- Added here left-to-right, top-to-bottom -->

Author: Should be in the format of:

1. Feature (couple of words max)
   One-liner about said feature 
   
  See [Dask site](https://www.dask.org/) for an example

Author: Suggestion "Power of Dask" -> "Dask powered"

Reviewer: I'm done updating the copy!

> Suggestion "Power of Dask" -> "Dask powered"
@trallard Oh, nice! I went with "Powerd by Dask", but I also like "Dask powered" -- feel free to select the one you prefer!

Author: I like Dask powered best!

Author: Integrated @pavithraes wording + some minor edits

1. Opinionated
   Designed with integrations and configurations selected from real-world experience, so that you can use it out-of-the-box for a variety of data science workloads.

1. Rooted in open source
   Developed with community in mind and under a BSD-3 license, we strive to contribute back to the upstream OSS projects wherever possible.

1. Collaboration-first
   Large teams can share work and iterate quickly with reproducible environments. Administrators can manage team resources effectively, all from the same platform.

1. Dask powered
   Nebari ships with Dask so you can scale your work to terabytes of data, leverage cloud instances with GPUs, and take advantage of adaptive scaling for managing costs.

1. Fully customizable
   Built with open source infrastructure and tools to give you complete flexibility over your deployment and fit your team's specific needs.

## Deploy anywhere

Try Nebari on your local machine or deploy it on your cloud of choice. Nebari is designed to be flexible, extensible, and vendor-agnostic.
Nebari can be seamlessly deployed to the major public cloud providers, including AWS, Azure, GCP, and Digital Ocean or to bare-metal HPC clusters.

> 

Reviewer: It sounds good to me!
